src/main/java/org/reaktivity/specification/mqtt/internal/MqttFunctions.java

```diff
@@ -179,22 +172,22 @@ public MqttDataExBuilder contentType(
             return this;
         }
 
-        public MqttDataExBuilder payloadFormat(
+        public MqttDataExBuilder format(
            String format)
         {
-            dataExRW.payloadFormat(p -> p.set(MqttPayloadFormat.valueOf(format)));
+            dataExRW.format(p -> p.set(MqttPayloadFormat.valueOf(format)));
             return this;
         }
 
-        public MqttDataExBuilder respTopic(
+        public MqttDataExBuilder responseTopic(
            String topic)
         {
-            dataExRW.respTopic(topic);
+            dataExRW.responseTopic(topic);
             return this;
         }
 
-        public MqttDataExBuilder respTopicCorrelationData(
-                String info)
+        public MqttDataExBuilder correlationInfo(
+           String info)
         {
             dataExRW.correlationInfo(c -> c.bytes(b -> b.set(info.getBytes())));
```

Comment:
Use `info.getBytes(StandardCharsets.UTF_8)` to avoid ambiguity of character set on different platforms.